### PR TITLE
style: :art: improve team members page styling and add Marton

### LINKED ISSUE #243: Team members page needs to be updated, it is weird

### PR DESCRIPTION
**Pull request by martonvago**
## Description

This PR improves the styling of the team members page and adds Marton.

Desktop:

Smaller screens:

Closes #243 

## Testing

- [ ] Yes
- [x] No, not needed (give a reason below)
- [ ] No, I need help writing them

Tested in the browser.

## Reviewer Focus

This PR only needs a quick review.

## Checklist

- [x] Tests accompany or reflect changes to the code
- [x] Tests ran and passed locally
- [x] Ran the linter and formatter
- [x] Build has passed locally
- [x] Relevant documentation has been updated

### CODE REVIEW

**Comment by martonvago** [comment]
Desktop:
team

Smaller screens:
team-small

File: about/team/styles.css [hunk 1]
```diff
@@ -0,0 +1,9 @@
+figcaption {
+  text-align: center;
+}
```
Comment (martonvago) [inline about/team/styles.css:3]: Do we want to center all captions everywhere?

Comment (signekb) [reply about/team/styles.css]: seedcase-project/seedcase-theme#27

**Review by lwjohnst86 — APPROVED** [review]
Very nice!!

**Pull request merged**